sentinel-transport/sentinel-transport-simple-http/src/main/java/com/alibaba/csp/sentinel/transport/command/http/HttpEventTask.java

```diff
@@ -190,56 +303,31 @@ private void closeResource(Closeable closeable) {
         }
     }
 
-    private void handleResponse(CommandResponse response, /*@NonNull*/ final PrintWriter printWriter,
-        /*@NonNull*/ final OutputStream rawOutputStream) throws Exception {
+    private <T> void handleResponse(CommandResponse<T> response, final PrintWriter printWriter) throws Exception {
         if (response.isSuccess()) {
             if (response.getResult() == null) {
-                writeOkStatusLine(printWriter);
+                writeResponse(printWriter, StatusCode.OK, null);
                 return;
             }
-            // Write 200 OK status line.
-            writeOkStatusLine(printWriter);
             // Here we directly use `toString` to encode the result to plain text.
             byte[] buffer = response.getResult().toString().getBytes(SentinelConfig.charset());
-            rawOutputStream.write(buffer);
-            rawOutputStream.flush();
+            writeResponse(printWriter, StatusCode.OK, new String(buffer));
         } else {
             String msg = SERVER_ERROR_MESSAGE;
             if (response.getException() != null) {
                 msg = response.getException().getMessage();
             }
-            badRequest(printWriter, msg);
+            writeResponse(printWriter, StatusCode.BAD_REQUEST, msg);
         }
     }
-
-    /**
-     * Write `400 Bad Request` HTTP response status line and message body, then flush.
-     */
-    private void badRequest(/*@NonNull*/ final PrintWriter out, String message) {
-        out.print("HTTP/1.1 400 Bad Request\r\n"
-            + "Connection: close\r\n\r\n");
-        out.print(message);
-        out.flush();
-        writtenHead = true;
-    }
-
-    /**
-     * Write `500 Internal Server Error` HTTP response status line and message body, then flush.
-     */
-    private void internalError(/*@NonNull*/ final PrintWriter out, String message) {
-        out.print("HTTP/1.1 500 Internal Server Error\r\n"
-            + "Connection: close\r\n\r\n");
-        out.print(message);
-        out.flush();
-        writtenHead = true;
-    }
-
-    /**
-     * Write `200 OK` HTTP response status line and flush.
-     */
-    private void writeOkStatusLine(/*@NonNull*/ final PrintWriter out) {
-        out.print("HTTP/1.1 200 OK\r\n"
-            + "Connection: close\r\n\r\n");
+    
+    private void writeResponse(PrintWriter out, StatusCode statusCode, String message) {
+        out.print("HTTP/1.0 " + statusCode.toString() + "\r\n"
+                + "Content-Length: " + (message == null ? 0 : message.length()) + "\r\n"
```

Comment:
The content length here is wrong when the message contains non-ASCII characters (e.g. Chinese characters). Maybe we should use `message.getBytes().length` instead?

Reply:
Oh you are right i forget to correct here.